src/main/scala/com/ubirch/kafka/metrics/PrometheusMetricsHelper.scala

```diff
@@ -1,24 +1,49 @@
 package com.ubirch.kafka.metrics
 
-import java.net.BindException
+import java.net.{ BindException, InetSocketAddress }
 
 import com.typesafe.scalalogging.LazyLogging
 import io.prometheus.client.exporter.HTTPServer
-import io.prometheus.client.hotspot.DefaultExports
+import io.prometheus.client.hotspot.{ BufferPoolsExports, ClassLoadingExports, DefaultExports, GarbageCollectorExports, MemoryAllocationExports, MemoryPoolsExports, StandardExports, ThreadExports, VersionInfoExports }
+import io.prometheus.client.CollectorRegistry
+import io.prometheus.jmx.JmxCollector
+import javax.management.MalformedObjectNameException
 
 import scala.annotation.tailrec
 
 object PrometheusMetricsHelper extends LazyLogging {
 
-  final private val maxAttempts = 3
+  /**
+   * Simple and default connection backed up by the default registry
+   * @param port the port number to start the server on
+   * @param maxAttempts the number of retries for when there is a Connection problem.
+   *                    If not provided, 3 attempts will be performed
+   * @return the Prometheus Http Server
+   */
+  def default(port: Int, maxAttempts: Int = 3): HTTPServer = create(port, defaultCollectorRegistry, maxAttempts)
```

Comment:
With this helper method, we have the possibility to use the default collector registry